docs_source_files/content/introduction/the_selenium_project_and_tools.de.md

```diff
@@ -3,120 +3,108 @@ title: "Das Seleniumprojekt und die Tools"
-have adopted Selenium for their browser-based testing,
-often replacing years-long efforts involving other proprietary tools.
-As it has grown in popularity, so have its requirements and challenges multiplied.
-
-As the web becomes more complicated
-and new technologies are added to websites,
-it's the mission of this project to keep up with them where possible.
-Being an open source project,
-this support is provided through the generous donation of time from many volunteers,
-every one of which has a “day job”.
-
-Another mission of the project is to encourage
-more volunteers to partake in this effort,
-and build a strong community
-so that the project can continue to keep up with emerging technologies
-and remain a dominant platform for functional test automation.
-
-
-### History
-
-When Selenium 1 was released in 2004,
-it was out of the necessity to reduce time spent
-manually verifying consistent behaviour in the front-end of a web application.
-It made use of what tools were available at the time 
-and relied heavily on the injection of JavaScript to the web page under test
-to emulate a user's interaction.
-
-Whilst JavaScript is a good tool to let you introspect the properties of the DOM
-and to do certain client-side observations that you would otherwise not be able to do,
-it falls short on the ability to naturally replicate a user's interactions
-as if the mouse and keyboard are being used.
-
-Since then, Selenium has grown and matured a lot,
-into a tool widely used by many&mdash;if not most&mdash;of
-the largest organisations around the world.
-Selenium has gone from a homebrewed test automation toolkit developed at Thoughtworks
-for a niché audience and a specific use case,
-to the world's _de facto_ browser automation library.
-
-Just as Selenium RC made use of the tools of the trade available at the time,
-[Selenium WebDriver]({{< ref "/webdriver/_index.md" >}}) drives that tradition on by taking
-the browser interaction part to the browser vendor's home turf 
-and asking them to take responsibility of the backend, browser-facing implementations.
-Recently this work has evolved into a W3C standardisation process
-where the goal is to turn the WebDriver component in Selenium
-into the _du jour_ remote control library for user agents.
+### Selenium steuert Webbrowsers
+
+_Selenium_ , ist Vieles, aber im Kern ist eine Sammlung von Werkzeugen um Webbrowser
+zu automatisieren. Es nutzt die besten Mechanismen die verfügbar sind um
+Browsersinstanzen zu steuern und Benutzereingaben zu simulieren.
+
+Es ermöglicht Aktionen von Benutzern zu simulieren;
+Texteingaben,
+Auswahl von Optionen in Drop-Downs, aktivieren von Checkboxen und
+anklicken von Links auf Webseiten.
+Es ermöglicht auch viele andere Möglichkeiten wie zum 
+Beispiel Mausbewegungen, beliebige JavaScript Codeausführungen
+und vieles mehr.
+
+In erster Linie wird Selenium für Front-End Testen von Websiten verwendet.
+Selenium ist im Grunde eine _'user agent' Bibliothek_. 
```

Comment:
user agent is placed inside quotes, is this required ?

Reply:
I think in German you tend to quote words which are in a different language, this is fine.

Reply:
Exactly, user agent could be translated to a German word, but often it makes no sense to translate an English word in IT to German.